internal/journal/voljournal.go

```diff
@@ -470,7 +470,7 @@ func reserveOMapName(ctx context.Context, monitors string, cr *util.Credentials,
 				continue
 			}
 
-			return "", err
+			return "", fmt.Errorf("failed to create object %w", err)
```

Comment:
Am thinking ca we disable Wrapcheck feels like this might cause an issue for error return and comparison, Lets hear it from others.

Reply:
Okay.

Reply:
Wrapping has my preference. We need to compare returned errors with `errors.Is()` if there is special handling of the errors needed. If comparing errors without `errors.Is()` is done, it needs to be corrected.